scheduling/priority_scheduling.py

```diff
@@ -0,0 +1,122 @@
+"""
+Implementation of Priority Scheduling
+
+https://en.wikipedia.org/wiki/Scheduling_(computing)#Priority_scheduling
+"""
+
+from operator import attrgetter
+
+"""
+
+Creating a class process which will have
+all required attributes
+
+"""
+
+
+class process:
+    def __init__(self, process_name, arrival_time, burst_time, priority):
+        self.process_name = process_name
+        self.arrival_time = arrival_time
+        self.burst_time = burst_time
+        self.priority = priority
+        self.waiting_time = None
+        self.turn_around_time = None
+
+
+n = int(input("Enter number of processes: ").strip())
```

Comment:
https://travis-ci.com/github/TheAlgorithms/Python/builds/178390061#L408 says that this line is hanging our automated testing as discussed in CONTRRIBUTING.md.  Please indent it under `if __name__ == "__main__":` so that Travis CI can test your submission.

Reply:
Added the line and indented the code coming after it